### PR TITLE
feature(REPORT-327626): Bold reports V6.1 Release Changes

### PR DESCRIPTION
**Pull request by Vishnu7101**
<table>
    <tr>
        <td>Feature description</td>
        <td>
To Update the Bold Reports - global public repo with the  Bold-Reports 6.1 Release Changes</td>
    </tr>
    <tr>
        <td>Analysis</td>
        <td> Analyzed about Bold reports V6.1 release changes</td>
    </tr>
    <tr>
        <td>Reason for not identifying earlier</td>
        <td>
            It's a new feature</td>
    </tr>
    <tr>
        <td>Is it a breaking issue?</td>
        <td>No, It's a new feature </td>
    </tr>
    <tr>
        <td>Solution description</td>
        <td>
<li> Update the version in package json to 6.1.34 in relevant to Bold-Reports 6.1 Release Changes
<li>To modify the files according to the latest changes made in Bold report V6.1
</td>
    </tr>
    <tr>
        <td>Areas affected and ensured</td>
        <td>NA</td>
    </tr>
    <tr>
        <td>Test cases</td>
        <td>NA</td>
    </tr>
    <tr>
        <td>Test bed sample locations</td>
        <td>VS & VS Code</td>
    </tr>
</table>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## About Bold Reports Global
 
-Bold Reports for JavaScript with language packs for the localization of [ReportViewer](https://www.boldreports.com/embedded-reporting/javascript-report-viewer?utm_source=github&utm_medium=backlinks) and [ReportDesigner](https://www.boldreports.com/embedded-reporting/javascript-report-designer?utm_source=github&utm_medium=backlinks) reporting widgets.
+Bold Reports for JavaScript with language packs for the localization of `ReportViewer` and `ReportDesigner` reporting widgets.
```
Comment (karthickthangasamy) [inline README.md]: revert this changes and move to our source

**Review by karthickthangasamy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Reverted

**Review by karthickthangasamy — APPROVED** [review]
